packages/plugin-relay/src/utils/connections.ts

```diff
@@ -3,6 +3,7 @@ import {
   encodeBase64,
   MaybePromise,
   PothosValidationError,
+  RemoveAwaitedProps,
```

Comment:
Since we know the return types of the connection helpers, we can remove the `MaybePromise` wrappers on the return type.